components/atomic/Tile/patterns/InvestigationTile/styles.js

```diff
@@ -63,52 +127,34 @@ export const Title = styled.div`
   }
 `;
 
-export const MixedLink = styled(BaseMixedLink)`
-  --Tile-bg-color: var(--turquoise85);
-  --Tile-color: var(--white);
-  --Tile-image-bg-color: var(--neutral20);
-
-  position: relative;
-  display: grid;
-  /* stylelint-disable declaration-block-no-redundant-longhand-properties */
-  grid-template-columns: 1fr;
-  grid-template-rows: ${fluidScale("234px", "164px")} 1fr;
-  grid-template-areas:
-    "image"
-    "title";
-  /* stylelint-enable declaration-block-no-redundant-longhand-properties */
-  align-content: start;
-  justify-items: center;
-  padding: 0;
-  border-radius: 16px;
-  color: var(--Tile-color);
-  text-decoration: none;
+export const EarlyAccessFlag = styled(EarlyAccess)`
+  position: absolute;
+  top: var(--Tile-image-margin-top);
+  right: 16px;
+  left: auto;
 
-  &[aria-disabled="true"] {
-    --Tile-bg-color: #6a6e6e;
-    --Tile-image-filter: grayscale(100%);
+  @media (max-width: ${BREAK_DESKTOP}) {
+    right: 0;
+    border-top-right-radius: 16px;
+  }
 
-    pointer-events: none;
-    cursor: default;
+  @media (max-width: ${BREAK_PHABLET}) {
+    top: 0;
+    right: auto;
+    left: 100px;
+  }
 
+  ${OffsetWrapper} {
     @media (max-width: ${BREAK_PHABLET}) {
-      --Tile-bg-color: transparent;
-      --Tile-color: #6a6e6e;
+      margin-left: -200%;
```

Comment:
I think this is a legit "css trick" kinda thing, but I didn't really test across browsers.  My "offset an element so that it can be positioned along its center axis" skills are a bit rusty.  Better more bulletproof way to accomplish this?